File: Selenium/Project/Project_Java/Activity4.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity4 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs/");

        // Get the second heading on the page
        String secondHeading = driver.findElement(By.xpath("(//h2)[1]")).getText();

        // Verify the website's second heading
        if (secondHeading.equals("Quia quis non")) {
            System.out.println("Second heading matches: " + secondHeading);
        } else {
            System.out.println("Second heading does not match: " + secondHeading);
        }

        // Close the browser
        driver.quit();
    }
}

File: Selenium/Project/Python_Project/Activity4.py
from selenium import webdriver

# Initialize the WebDriver
driver = webdriver.Chrome()

try:
    # Navigate to the website
    driver.get('https://alchemy.hguy.co/jobs/')
    
    # Get the second heading on the page
    second_heading = driver.find_element("xpath","//h2").text
    
    # Verify the second heading
    assert second_heading == "Quia quis non", "Second heading does not match"
    print("Second heading verification successful.")

finally:
    # Close the browser
    driver.quit()